tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -44,7 +44,7 @@
 #include "tiledb/sm/stats/stats.h"
 #include "tiledb/sm/storage_manager/storage_manager.h"
 #include "tiledb/sm/tile/tile.h"
-#include "tiledb/sm/tile/tile_io.h"
+#include "tiledb/sm/tile/generic_tile_io.h"
```

Comment:
Include statements should be ordered alphabetically. I do not think this is enforced by the format check, but we try to follow this rule the best that we can.